src/frontend/components/pages/wallet/index.tsx

```diff
@@ -258,6 +266,52 @@ const Wallet = ({ principal }: WalletProps) => {
         toast.warn("Failed to list");
       });
   };
+  const { call: burn } = bip721LedgerActor.useUpdateCall({
+    functionName: "icrcX_burn",
+  });
+
+  const triggerDelete = (intPropId: bigint) => {
```

Comment:
We should avoid duplicating the same function. What do you think about isolating this function in a hook in single file?